feat: UTC-106: ActionButton disabled functionality

mcanu opened the pull request:
<!--

This description MUST be filled out for a PR to receive a review. Its primary purposes are:

 - to enable your reviewer to review your code easily, and
 - to convince your reviewer that your code works as intended.

Some pointers to think about when filling out your PR description:
 - Reason for change: Description of problem and solution
 - Screenshots: All visible changes should include screenshots.
 - Rollout strategy: How will this code be rolled out? Feature flags / env var / other
 - Testing: Description of how this is being verified
 - Risks: Are there any known risks associated with this change, eg to security or performance?
 - Reviewer notes: Any info to help reviewers approve the PR
 - General notes: Any info to help onlookers understand the code, or callouts to significant portions.

You may use AI tools such as Copilot Actions to assist with writing your PR description (see https://docs.github.com/en/copilot/using-github-copilot/using-github-copilot-for-pull-requests/creating-a-pull-request-summary-with-github-copilot); however, an AI summary isn't enough by itself. You'll need to provide your reviewer with strong evidence that your code works as intended, which requires actually running the code and showing that it works.

-->

mcanu committed:
Added ActionButton disabled functionality

hakan458 commented on label_studio/data_manager/actions/__init__.py:
```diff
@@ -56,6 +56,10 @@ def get_all_actions(user, project):
             action['dialog']['form'] = form_generator(user, project)
 
+        disabled_generator = action.get('disabled')
+        if callable(disabled_generator):
```
how does this work exactly? isn't disabled a boolean?

hakan458 replied:
Ah I see in LSE there are functions that evaluate to a boolean

mcanu replied:
Its the same pattern we use for `form`. It can be an already defined boolean or a callable that returns a boolean.

nass600 reviewed (approved):
Nice implementation from the FE perspective @mcanu

mcanu committed:
Disabling tooltip

mcanu committed:
Merge branch 'develop' into fb-utc-106

mcanu commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/15619089347)

mcanu committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/15619089347

mcanu committed:
Merge branch 'develop' into 'fb-utc-106'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/15619089347

mcanu commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/15620607076)

mcanu committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/15620607076

mcanu committed:
Merge branch 'develop' into 'fb-utc-106'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/15620607076

mcanu commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/15636178042)

mcanu committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/15636178042

mcanu committed:
Merge branch 'develop' into 'fb-utc-106'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/15636178042

mcanu commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/15637585481)

mcanu committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/15637585481
